libsrc/_DEVELOPMENT/math/float/am9511/c/8085/am9511_asinh.asm

```diff
@@ -185,41 +185,37 @@
 
 ; Function am9511_asinh flags 0x00000208 __smallc __z88dk_fastcall 
 ; double am9511_asinh(double x)
-; parameter 'double x' at sp+2 size(6)
+; parameter 'double x' at sp+2 size(4)
 	C_LINE	8,"am9511_asinh.c::am9511_asinh::0::0"
 ._am9511_asinh
-	call	dpush
-	ld	hl,0	;const
-	add	hl,sp
-	call	dload
+	push	de
+	push	hl
 	call	_fabs
 	call	_mul2
-	call	dpush
-	ld	hl,6	;const
+	push	de
+	push	hl
+	ld	hl,4	;const
 	add	hl,sp
-	call	dload
+	call	l_glong
 	call	_sqr
-	call	dpush
-	ld	hl,1	;const
-	call	l_int2long_u_float
-	call	dadd
+	push	de
+	push	hl
+	ld	hl,0	;const
+	ld	de,16256
+	call	l_f32_add
 	call	_sqrt
-	call	dpush
-	ld	hl,12	;const
+	push	de
+	push	hl
+	ld	hl,8	;const
 	add	hl,sp
-	call	dload
+	call	l_glong
```

Comment:
Shouldn't this be `call fabs` since `fabs()` is a sccz80 library function?

Reply:
TBH, I'm playing with the [makefile](https://github.com/z88dk/z88dk/blob/8827c31cb7847d249f71d274748766590664330f/libsrc/_DEVELOPMENT/math/float/am9511/c/Makefile#L14) to try to get something that looks sensible for use with sccz80/classic/8085. This was the first cut.

At the moment it is compiling against newlib. I know that's incorrect, but I haven't got the right magic to pull in the classic header files and make sure it is using sccz80 to compile. Hints / suggestions on how to get something sensible are welcome. `make` is not my friend.

Reply:
Ah right, yeah, `zcc +test -clib=8085` should do it for you.